src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -22,14 +22,17 @@ class Focuser(AbstractSerialFocuser):
     as they are marked with the decorator @abstractmethod, we have to override them.
     """
 
-    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', vendor_id=None,
-                 product_id=None, *args, **kwargs):
+    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', port=None,
+                 vendor_id=None, product_id=None, *args, **kwargs):
```

Comment:
This doesn't seem to work either:
```
In [4]: f = Focuser(port="/dev/ttyUSB0")
I 06-24 04:28:01.883 Creating PanDB panoptes
D 06-24 04:28:01.887 Focuser created: Astromechanics Focuser on /dev/ttyUSB0
C 06-24 04:28:01.894 Could not open /dev/ttyUSB0!
E 06-24 04:28:01.896 Error connecting to Astromechanics Focuser on /dev/ttyUSB0: [Errno 6] could not open port /dev/ttyUSB0: [Errno 6] No such device or address: '/dev/ttyUSB0'
<ipython-input-1-4cba6ca25adf>:38: UserWarning: Error connecting to Astromechanics Focuser on /dev/ttyUSB0: [Errno 6] could not open port /dev/ttyUSB0: [Errno 6] No such device or address: '/dev/ttyUSB0'
  warn(message)
D 06-24 04:28:01.902 Initializing Astromechanics Focuser
```
Despite:
```
In [9]: ls -l /dev | grep "ttyUSB0"
crw-rw---- 1 root      dialout 188,   0 Jun 24 04:18 ttyUSB0
```
I note this is how we were creating the focuser instances prior to the recent changes.